modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -525,3 +527,42 @@ function editConsentStatusFields($db, $user)
         }
     }
 }
+
+/**
+ * Handles the updating of candidate's date of death.
+ *
+ * @param Database $db   database object
+ * @param User     $user user object
+ *
+ * @throws DatabaseException
+ *
+ * @return void
+ */
+function editCandidateDOD(\Database $db, \User $user): void
+{
+    $candID       = new CandID($_POST['candID']);
+    $dod          = new DateTime($_POST['dod']);
+    $strippedDate = null;
+    $dodString    = null;
+
+    if (!$dod) {
+        throw new \LorisException('Date not valid.');
+    }
+
+    if (!empty($dod)) {
+        $config    = \NDB_Config::singleton();
+        $dodFormat = $config->getSetting('dodFormat');
+        if ($dodFormat === 'YM') {
+            $strippedDate = $dod->format('Y-m') . "-01";
```

Comment:
```suggestion
            $strippedDate = $dod->format('Y-m-01');
```

I think that this is possible, just for future reference.